packages/vue/src/components/SpCrudRead/SpCrudRead.vue

```diff
@@ -92,20 +106,32 @@ export default defineComponent({
 
     // computed
     let address = computed(() => $s.getters['common/wallet/address'])
-    let posts = computed(
-        () => $s.state[props.storeName]
+    let itemFields = computed(() => $s.getters[props.storeName + '/getTypeStructure'](props.itemName))
+    let items = computed(
+      () => {
+        const itemData = $s.state[props.storeName][props.itemName + 's']
+        const queryKey = Object.keys(itemData)[0]
+        if (queryKey && itemData[queryKey]) return itemData[queryKey][props.itemName]
+        return []
+      }
     )
 
-    onBeforeMount(async () => {
-      await $s.dispatch(`${props.storeName}/QueryPosts`, {
-        options: { subscribe: true },
-      })
+    $s.dispatch(`${props.storeName}${props.commandName}`, {
```

Comment:
Is this right? Doesn't seem to be...

Reply:
Also we are missing a dispatch to query all items as in [here](https://github.com/tendermint/vue/blob/8592c94e43592b17c7367606ace161d80a9fff74/packages/vue/src/components/SpTypeList/SpTypeList.vue#L146)

Reply:
What's wrong with this?